libs/transactions/src/lib/transactions-journey.module.ts

```diff
@@ -69,7 +77,7 @@ export class TransactionsJourneyModule {
     return {
       ngModule: TransactionsJourneyModule,
       providers: [
-        provideRoutes([route || defaultRoute]),
+        provideRoutes([...(defaultRoute || route)]),
```

Comment:
```suggestion
        provideRoutes([...(route || defaultRoute)]),
```

Reply:
But why?

Reply:
defaultRoute is an array of routes now

Reply:
the logic is that we use a custom `route` and have a fallback to `defaultRoute`. In your change we do the opposite therefore we always receive `defaultRoute` event if `route` is set